project/accounts/urls.py

```diff
@@ -35,3 +38,25 @@
     ),
     path('setup/', ProfileSetupView.as_view(), name="accounts_profile_setup"),
 ]
+
+
+# REST endpoints
+router = DefaultRouter(trailing_slash=False)
+router.register(r"accounts", api.ProfileViewSet)
```

Comment:
This endpoint should be prefixed "profiles" since it is returning Profile objects.
```suggestion
router.register(r"profiles", api.ProfileViewSet)
```